models/pathology_nuclick_annotation/configs/train.json

```diff
@@ -62,7 +62,7 @@
                     ]
                 },
                 {
-                    "_target_": "SplitLabeld",
+                    "_target_": "monai.apps.nuclick.transforms.SplitLabeld",
```

Comment:
Hi @ericspod, seems after this [PR](https://github.com/Project-MONAI/MONAI/pull/8113), all this transforms under monai.apps.nuclick can not't be imported correctly.
Does this expected?
```
[2024-09-27T08:56:23.320Z] ======================================================================
[2024-09-27T08:56:23.320Z] ERROR: test_train_eval_config_0 (test_pathology_nuclei_classification.TestNucleiCls)
[2024-09-27T08:56:23.320Z] ----------------------------------------------------------------------
[2024-09-27T08:56:23.320Z] Traceback (most recent call last):
[2024-09-27T08:56:23.320Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/parameterized/parameterized.py", line 620, in standalone_func
[2024-09-27T08:56:23.320Z]     return func(*(a + p.args), **p.kwargs, **kw)
[2024-09-27T08:56:23.320Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/ci/unit_tests/test_pathology_nuclei_classification.py", line 108, in test_train_eval_config
[2024-09-27T08:56:23.320Z]     check_workflow(trainer, check_properties=True)
[2024-09-27T08:56:23.320Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/ci/unit_tests/utils.py", line 116, in check_workflow
[2024-09-27T08:56:23.320Z]     check_result = workflow.check_properties()
[2024-09-27T08:56:23.320Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/workflows.py", line 379, in check_properties
[2024-09-27T08:56:23.320Z]     ret = super().check_properties()
[2024-09-27T08:56:23.320Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/workflows.py", line 225, in check_properties
[2024-09-27T08:56:23.320Z]     return [n for n, p in self.properties.items() if p.get(BundleProperty.REQUIRED, False) and not hasattr(self, n)]
[2024-09-27T08:56:23.320Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/workflows.py", line 225, in <listcomp>
[2024-09-27T08:56:23.320Z]     return [n for n, p in self.properties.items() if p.get(BundleProperty.REQUIRED, False) and not hasattr(self, n)]
[2024-09-27T08:56:23.320Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/workflows.py", line 175, in __getattr__
[2024-09-27T08:56:23.321Z]     return self._get_property(name=name, property=self.properties[name])
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/workflows.py", line 420, in _get_property
[2024-09-27T08:56:23.321Z]     return self.parser.get_parsed_content(id=prop_id) if prop_id is not None else None
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/config_parser.py", line 290, in get_parsed_content
[2024-09-27T08:56:23.321Z]     return self.ref_resolver.get_resolved_content(id=id, **kwargs)
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/reference_resolver.py", line 193, in get_resolved_content
[2024-09-27T08:56:23.321Z]     return self._resolve_one_item(id=id, **kwargs)
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/reference_resolver.py", line 163, in _resolve_one_item
[2024-09-27T08:56:23.321Z]     self._resolve_one_item(id=d, waiting_list=waiting_list, **kwargs)
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/reference_resolver.py", line 163, in _resolve_one_item
[2024-09-27T08:56:23.321Z]     self._resolve_one_item(id=d, waiting_list=waiting_list, **kwargs)
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/reference_resolver.py", line 163, in _resolve_one_item
[2024-09-27T08:56:23.321Z]     self._resolve_one_item(id=d, waiting_list=waiting_list, **kwargs)
[2024-09-27T08:56:23.321Z]   [Previous line repeated 1 more time]
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/reference_resolver.py", line 171, in _resolve_one_item
[2024-09-27T08:56:23.321Z]     self.resolved_content[id] = item.instantiate() if kwargs.get("instantiate", True) else item
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/bundle/config_item.py", line 292, in instantiate
[2024-09-27T08:56:23.321Z]     return instantiate(modname, mode, **args)
[2024-09-27T08:56:23.321Z]   File "/home/jenkins/agent/workspace/MONAI-zoo-premerge/model_zoo_venv/lib/python3.10/site-packages/monai/utils/module.py", line 196, in instantiate
[2024-09-27T08:56:23.321Z]     raise ModuleNotFoundError(f"Cannot locate class or function path: '{__path}'.")
[2024-09-27T08:56:23.321Z] ModuleNotFoundError: Cannot locate class or function path: 'SplitLabeld'.
```

Reply:
As we discussed this isn't correct, I'm going to undo some changes here to correct this for now: https://github.com/Project-MONAI/MONAI/pull/8118

Reply:
revert the change, thanks.